windows-release/uploadrelease.ps1

```diff
@@ -87,19 +88,28 @@ $dirs = gci "$build" -Directory
 if ($embed) {
     $dirs = ($dirs, (gi $embed)) | %{ $_ }
 }
+if ($sbom) {
+    $dirs = ($dirs, $sbom)
+}
 
 foreach ($a in $dirs) {
     "Uploading files from $($a.FullName)"
     pushd "$($a.FullName)"
     $exe = gci *.exe, *.exe.asc, *.zip, *.zip.asc
     $msi = gci *.msi, *.msi.asc, *.msu, *.msu.asc
+    $spdx_json = gci *.spdx.json
     popd
 
     if ($exe) {
         & $pscp -batch -hostkey $hostkey -noagent -i $keyfile $exe.FullName "$user@${server}:$d"
         if (-not $?) { throw "Failed to upload $exe" }
     }
 
+    if ($spdx_json) {
```

Comment:
This probably isn't right, the SBOM directory contains more than one file, but I believe other artifacts are like that too so I duplicated what was done for exe?